packages/flutter/test/widgets/framework_test.dart

```diff
@@ -1756,6 +1799,95 @@ The findRenderObject() method was called for the following element:
   });
 }
 
+class _AsyncInitState extends StatefulWidget {
```

Comment:
For example, suppose I want to test "async dispose should trigger throw". Then the initState must *not* be async. Otherwise, we will see error from initState already.